CHANGELOG.md

```diff
@@ -10,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - **CUMULUS-1212**
   - `@cumulus/post-to-cmr` will now fail if any granules being processed are missing a metadata file. You can set the new config option `skipMetaCheck` to `true` to pass post-to-cmr without a metadata file.
+- **CUMULUS-1264**
+  - The deployment configuration for the nested API stacks has been substantially refactored. **As a result, you must deleting your existing app stack before deploying this version of Cumulus.**
```

Comment:
s/you must deleting your existing app stack/ you should delete your stack before deploying/  ?

Reply:
I think you have to do it. If not, then the deployment will fail because the nested stacks now manage some resources that were previously in the parent stack and CF will complain about them already existing when it tries to deploy